etc/services/osf-localhost.json

```diff
@@ -12,5 +12,25 @@
         "given-names", "family-name"
       ]
     ]
+  },
+  "properties" : {
+    "@class" : "java.util.HashMap",
+    "title" : {
+      "@class" : "org.jasig.cas.services.DefaultRegisteredServiceProperty",
+      "values" : [
+        "java.util.HashSet", [
+          "Open&nbsp;Science&nbsp;Framework",
```

Comment:
Is there a reason we cannot use CSS on the page where this is displayed so we don't have to encode `&nbsp;`?